### PR TITLE
Add Story Pill to Story Cards 

### PR DESCRIPTION
**Pull request by andrewyguo**
## Notion ticket link
[Add Story Pill](https://www.notion.so/uwblueprintexecs/Add-language-pill-to-all-story-cards-ce7044a46e8944a1891aa613f9b83b0c)

## Implementation description
* Add a file in /frontend/src/utils to convert language string to Title Case 
* Remove conditional that storyTranslationId needs to be defined before showing story language pill 

## Steps to test
1. Log in as a user and see if story language pill is displayed 

## What should reviewers focus on?
* Ensure that the 4 edge cases are displayed properly (ENGLISH_US, ENGLISH_UK, ENGLISH_INDIA, ASANTE_TWI) 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters:  `docker exec -it <backend-container-id> /bin/bash -c "black . && isort --profile black ."`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/homepage/StoryCard.tsx [hunk 1]
```diff
@@ -155,7 +155,13 @@ const StoryCard = ({
           <div className="story-card-tags">
             {/* TODO: make tags that reflect actual state */}
             <Tag displayText={`Level ${level}`} />
-            {storyTranslationId && <Tag displayText={`${language}`} />}
+            {!storyTranslationId && storyId && (
+              <Tag
+                displayText={`${
+                  language[0] + language.substring(1).toLowerCase()
```
Comment (gaoxk) [inline frontend/src/components/homepage/StoryCard.tsx:161]: I don't think that'll look correct for ASANTE_TWI,  ENGLISH_UK, ENGLISH_US, or ENGLISH_INDIA. Can you make a frontend util function for converting languages to title case? In that function, you can deal with the 4 edge cases but otherwise use this existing line of code. Feel free to make a new file in `frontend/src/utils`

**Review by gaoxk — CHANGES_REQUESTED** [review]

File: frontend/src/components/homepage/StoryCard.tsx [hunk 2]
```diff
@@ -155,7 +156,7 @@ const StoryCard = ({
           <div className="story-card-tags">
             {/* TODO: make tags that reflect actual state */}
```
Comment (gaoxk) [inline frontend/src/components/homepage/StoryCard.tsx:157]: you can remove this TODO; Russ gave the 👍 for these tags

File: frontend/src/utils/StoryCardUtils.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// Change language string to title case
```
Comment (gaoxk) [inline frontend/src/utils/StoryCardUtils.ts:1]: nit: name the file something like `languages `- this utility wont necessarily only be used by the `StoryCard`

File: frontend/src/utils/StoryCardUtils.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+function changeLanguageToTitleCase(language: string) {
```
Comment (gaoxk) [inline frontend/src/utils/StoryCardUtils.ts:2]: nit: maybe a name like `convertLanguageTitleCase` works better - `change` can imply modifying the passed in param

**Review by gaoxk — CHANGES_REQUESTED** [review]

**Review by gaoxk — APPROVED** [review]
🚢
